### PR TITLE
refactor: move `panes` to desktop `components` folder

### PR DESCRIPTION
**Pull request by maxwellmattryan**
## Summary

> Please summarize your changes, describing **what** they are and **why** they were made.

These are desktop-specific components that don't belong in shared.

## Changelog

```
- Move panes over to desktop components folder
- Adjust all import paths in components that use a pane
- Add an extra path alias for specific UI imports (mobile and desktop)
```

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [x] Linux
    -   [ ] Windows
-   **Mobile**
    -   [ ] iOS
    -   [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

Navigate throughout the app where there are panes, making sure no errors occur.

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [x] I have followed the contribution guidelines for this project
-   [x] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/desktop/views/dashboard/developer/Developer.svelte [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
         CreateAliasButton,
-        Pane,
         RefreshTokenMetadataButton,
         Text,
         TestDeepLinkButton,
-    } from 'shared/components'
+    } from '@ui'
+    import { Pane } from '@components'
     import features from '@features/features'
+    import { selectedAccount } from '@core/account/stores'
```
Comment (begonaalvarezd) [inline packages/desktop/views/dashboard/developer/Developer.svelte:13]: do we need to specify `/stores` now?

File: packages/desktop/views/dashboard/governance/views/GovernanceDashboardView.svelte [hunk 2]
```diff
@@ -1,7 +1,8 @@
 <script lang="ts">
-    import { Pane, Proposals, ProposalsDetails, VotingPower, Illustration, Text, FontWeight } from '@ui'
+    import { Proposals, ProposalsDetails, VotingPower, Illustration, Text, FontWeight } from '@ui'
+    import { Pane } from '@components'
     import { localize } from '@core/i18n'
-    import { registeredProposalsForSelectedAccount } from '@contexts/governance'
+    import { registeredProposalsForSelectedAccount } from '@contexts/governance/stores'
```
Comment (begonaalvarezd) [inline packages/desktop/views/dashboard/governance/views/GovernanceDashboardView.svelte:5]: like here also, why do we need `/stores`?

**Review by begonaalvarezd — CHANGES_REQUESTED** [review]
looking great! just added some questions 🙏🏼

File: packages/desktop/views/dashboard/developer/Developer.svelte (hunk 1 above)
Comment (maxwellmattryan) [reply packages/desktop/views/dashboard/developer/Developer.svelte:13]: Yes it helps out with circular dependency issues. David and I have started looking into a solution, hopefully we can try it soon

File: packages/shared/components/panes/index.js [hunk 3]
```diff
@@ -1 +0,0 @@
-export { default as Pane } from './Pane.svelte'
```
[the remark this answers is not in the document]
Comment (maxwellmattryan) [reply packages/shared/components/panes/index.js:1]: It's an atom in the sense that it's really simple with no logic inside but it usually houses more complex components. I can move it to `atoms`

**Review by begonaalvarezd — APPROVED** [review]
🚀

**Pull request merged**